### PR TITLE
Task 1 dynamic array

### PR DESCRIPTION
**Pull request by TarasOheruk**

### CODE REVIEW

File: dynamicArray.cpp [hunk 1]
```diff
@@ -1,20 +1,66 @@
...
-// Tips for Getting Started: 
-//   1. Use the Solution Explorer window to add/manage files
-//   2. Use the Team Explorer window to connect to source control
-//   3. Use the Output window to see build output and other messages
-//   4. Use the Error List window to view errors
-//   5. Go to Project > Add New Item to create new code files, or Project > Add Existing Item to add existing code files to the project
-//   6. In the future, to open this project again, go to File > Open > Project and select the .sln file
+using namespace std;
+
+class List {
```
Comment (bodyo) [inline dynamicArray.cpp:14]: please consider using only the functions - as we discussed earlier

File: dynamicArray.cpp [hunk 2]
```diff
@@ -1,20 +1,66 @@
-// dynamicArray.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
+//Created by Oheruk Taras
+/*
+    TODO:
+    1.���������� � size (��� ����� �������� �� ���)
```
Comment (bodyo) [inline dynamicArray.cpp:4]: please do not use UNICODE

File: .vs/dynamicArray/v17/Browse.VC.db
Comment (bodyo) [inline .vs/dynamicArray/v17/Browse.VC.db:1]: please add `.vs/` folder to gitignore and remove all files in this folder from git history

File: Debug/dynamicArray.ilk
Comment (bodyo) [inline Debug/dynamicArray.ilk:1]: same for `Debug/` as for `.vs/`

File: dynamicArray.cpp [hunk 3]
```diff
@@ -1,20 +1,66 @@
...
+    int capacity;
+
+public:
+    List() {
+        size = 0;
+        capacity = 5;
+        array = new int[capacity];
+    }
+
+    void add(int index, int value) {
```
Comment (bodyo) [inline dynamicArray.cpp:27]: the API for dynamic vector should have following functions:
* push_back - adds element to the end of array.
* push_front - adds element to the start of array.
* pop_back - removes element from the end of array.
* pop_front - removes element from the start of array.
* clear - clears dynamic array.
* remove(int index) removes element from array (size should become size-1)

Please use this API for reference: https://en.cppreference.com/w/cpp/container/vector `Modifiers` section

File: x64/Debug/dynamicArray.tlog/CL.read.1.tlog
Comment (bodyo) [inline x64/Debug/dynamicArray.tlog/CL.read.1.tlog:1]: same as for `.vs/` folder

File: .vs/dynamicArray/v17/Browse.VC.db
Comment (TarasOheruk) [reply .vs/dynamicArray/v17/Browse.VC.db:1]: added .gitignore and removed these files from cache

File: Debug/dynamicArray.ilk
Comment (TarasOheruk) [reply Debug/dynamicArray.ilk:1]: added .gitignore and removed these files from cache

File: x64/Debug/dynamicArray.tlog/CL.read.1.tlog
Comment (TarasOheruk) [reply x64/Debug/dynamicArray.tlog/CL.read.1.tlog:1]: added .gitignore and removed these files from cache

**Comment by bodyo** [comment]
please also read this article about best "commit messages practice" 
https://dev.to/ishanmakadia/git-commit-message-convention-that-you-can-follow-1709

**Pull request closed without merging**